packages/http-client-csharp/generator/TestProjects/CadlRanch/http/authentication/http/custom/src/Generated/AuthenticationHttpCustomModelFactory.cs

```diff
@@ -1,11 +0,0 @@
-// <auto-generated/>
-
-#nullable disable
-
-namespace Authentication.Http.Custom.Models
-{
-    public static partial class AuthenticationHttpCustomModelFactory
```

Comment:
This gets deleted because the only model in the library was deleted as it was just used internally.